src/test/kotlin/pt/up/fe/ni/website/backend/controller/ProjectControllerTest.kt

```diff
@@ -241,6 +262,7 @@ internal class ProjectControllerTest @Autowired constructor(
                     content().contentType(MediaType.APPLICATION_JSON),
                     jsonPath("$.title").value(testProject.title),
                     jsonPath("$.description").value(testProject.description),
+                    jsonPath("$.hallOfFame.length()").value(1),
```

Comment:
I think here you could not only verify the length of the list but also that the correct element was added (ex. by checking the email)